### ISSUE TITLE
Active tasks component

### ISSUE THREAD

**Issue by sneha-pai** [department-of-veterans-affairs/caseflow#8141]

**Comment by sneha-pai** [comment]
### The scope below need to be broken down progressively into technical tickets:

- A case will be defined as  "currently active" if it is assigned to an individual or team(except colocated team) and is not on hold due to a blocking task.

- If a case is on hold for any user/team (due to a blocking task), it will appear in Case Timeline, with a grey dot (incomplete) rather than a green checkmark dot.

- Cases on-hold for the colocated team will not appear in the case timeline.

- Once a case is marked complete, it is no longer currently active, and it will now appear as a green-check-marked case in Case Timeline.

- If a colocated individual places a case on hold (with a window of time), and that task has no child task, that case will continue to appear as Currently Active.


#### Scenario: No Task assigned to current user (viewer just sees task list, but no "actions" dropdown.
![current-task-viewer-not-assigned](https://user-images.githubusercontent.com/31224257/49607317-96b01200-f963-11e8-8b98-63e359af6511.png)


#### Scenario: One task assigned to current user (colocated individual)
![current-task-single-task-assigned-to-colocated-user](https://user-images.githubusercontent.com/31224257/49608674-3fac3c00-f967-11e8-8614-567fadf1db55.png)

#### Scenario: When colocated is done with their, the attorney's task is now unblocked (viewing as attorney)
![current-task-single-task-assigned-to-current-user-attorney-unblocked](https://user-images.githubusercontent.com/31224257/49607326-a0397a00-f963-11e8-9db4-18ef6a2ee33f.png)

#### If colocated were to assign this task to another team, the colocated task would be on hold and blocked (viewing as AOD individual)
![current-task-single-task-assigned-to-aod-individual](https://user-images.githubusercontent.com/31224257/49607330-a3cd0100-f963-11e8-85b0-8cf1f5f0abd4.png)

#### Scenario: Two tasks assigned to current user (default view)
![current-task-multiple-tasks-assigned-to-current-user-task-instructions-closed](https://user-images.githubusercontent.com/31224257/49607337-a7f91e80-f963-11e8-8588-663d4083072d.png)


#### Scenario: Two tasks assigned to current user + Task instructions are open
![current-task-multiple-tasks-assigned-to-current-user-task-instructions-open](https://user-images.githubusercontent.com/31224257/49607349-afb8c300-f963-11e8-96ad-a1d26213cccf.png)




Please see https://app.mural.co/t/workqueue2001/m/workqueue2001/1543851111020/bbd6df5b95ca9a9ad179cd991d40c9314f35c929 for more details about the active-task logic.
